.github/scripts/shared/utils.ts

```diff
@@ -39,12 +39,34 @@ export function getCurrentDateFormatted(): string {
 
 // This mapping is used to know what planning repo is used for each code repo
 export const codeRepoToPlanningRepo: { [key: string]: string } = {
-  "metamask-extension": "MetaMask-planning",
-  "metamask-mobile": "mobile-planning"
-}
+  'metamask-extension': 'MetaMask-planning',
+  'metamask-mobile': 'mobile-planning',
+};
```

Comment:
This change seems to indicate that we are not linting this file, but where is that ignore happening?